config/locales/es.yml

```diff
@@ -40,17 +40,17 @@ es:
 
   search:
     form_placeholder: Busca en los archivos…
-    search_results: 'Search results for: %{query}'
+    search_results: 'Resultados de la búsqueda de: %{query}'
```

Comment:
would the second "de" change based on the value of `%{query}`?

If so, maybe we just translate "Search results:" rather than "Search results for:"? What do you think?

Reply:
I've updated "search_results" to "Resultados de la búsqueda: %{query}".